client/src/pages/Project.tsx

```diff
@@ -291,7 +307,7 @@ export default function Project() {
               <button
                 type="submit"
                 className={`bg-blue-500 hover:bg-blue-600 transition-colors text-slate-100 px-4 rounded-lg mt-4 flex items-center justify-center min-w-[10rem] disabled:bg-blue-400`}
-                disabled={createJoinReqMutation.isLoading}
+                disabled={false}
```

Comment:
I think you forgot to change this for testing purposes, make sure this has the mutation loading conditional